packages/@okta/vuepress-site/use_cases/inline_hooks/registration_hook/registration_hook/index.md

```diff
@@ -0,0 +1,249 @@
+---
+title: Registration Inline Hook
+excerpt: Customize handling of user registration requests in Self-Service Registration
+---
+
+# Registration Inline Hook
+
+<ApiLifecycle access="ea" />
+
+This page provides reference documentation for:
+
+- JSON objects contained in the outbound request from Okta to your external service
+
+- JSON objects you can include in your response
+
+This information is specific to the Registration Inline Hook, one type of inline hook supported by Okta.
+
+## See Also
+
+For a general introduction to Okta inline hooks, see [Inline Hooks](/use_cases/inline_hooks/).
+
+For information on the API for registering external service endpoints with Okta, see [Inline Hooks Management API](/docs/api/resources/inline-hooks).
+
+For steps to enable this inline hook, see below, [Enabling a Registration Inline Hook](#enabling-a-registration-inline-hook-for-self-service-registration).
+
+## About
+
+The Okta Registration Inline Hook allows you to integrate your own custom code into Okta's [Self-Service Registration](https://help.okta.com/en/prod/Content/Topics/Directory/eu-self-service.htm) flow. The hook is triggered after Okta receives the registration request but before the user is created. Your custom code can:
+
+- Set or override the values that will be populated in attributes of the user's Okta profile
+- Allow or deny the registration attempt, based on your own validation of the information the user has submitted
+
+## Objects in the Request from Okta
+
+The outbound call from Okta to your external service includes the following objects in its JSON payload:
+
+### data.userProfile
+
+This object contains name-value pairs for each attribute supplied by the user in the Self-Service Registration form, except for the following:
+
+- the `password` field
+- any fields corresponding to user profile attributes marked as sensitive in your Okta user schema
+
+Using the `com.okta.user.profile.update` commands you send in your response, you can modify the values of the attributes, or add other attributes, before the values are assigned to the Okta user profile that will be created for the registering user.
+
+You can only set values for profile fields which already exist in your Okta user profile schema: Registration Inline Hook functionality can only set values, it cannot create new fields.
```

Comment:
I would avoid a comma splice here. Maybe use a dash or semicolon instead.

Reply:
Updated to use semicolon.